modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -11,22 +11,16 @@ The saved states are listed by the ID of the user who made the change, time, dat
 
 Being able to view a saved state of a project is useful if you want to:
```

Comment:
Why does this sentence say "to _view_ a saved state"? The topic is about restoring a saved state of a project.

Reply:
Great catch, not sure why I had it that way in the first place.

Reply:
Also, any thoughts on whether or not this is accurate?  'Retrieve previous values of connector configurations'. If not, what am I missing here?